core/src/test/java/org/apache/calcite/test/TableFunctionTest.java

```diff
@@ -105,6 +105,24 @@
     }
   }
 
+  @Test public void testTableFunctionWithArrayParameter() throws SQLException {
```

Comment:
may be add a case for map type parameter?

Reply:
I think array is enough, just like what the commit describes.